### PR TITLE
Review of Feature US-0000028

### PR DESCRIPTION
**Pull request by swongcopado**
Please review this and approve

### CODE REVIEW

File: force-app/main/default/classes/Story22Apex.cls
Comment (stevenckwong) [inline force-app/main/default/classes/Story22Apex.cls:1]: Greeting should check for user's language before returning. Right now it assumes everyone speaks English

File: force-app/main/default/classes/Story22ApexTest.cls
Comment (stevenckwong) [inline force-app/main/default/classes/Story22ApexTest.cls:1]: Assumes English. Need to think if we need to handle user locale.

**Review by stevenckwong — COMMENTED** [review]
See reviews on individual files

File: force-app/main/default/classes/Story22ApexTest.cls
Comment (swongcopado) [reply force-app/main/default/classes/Story22ApexTest.cls:1]: For this iteration, business has told us to assume its English but noted this down as something we need to look at in the next iteration

File: force-app/main/default/classes/Story22Apex.cls
Comment (swongcopado) [reply force-app/main/default/classes/Story22Apex.cls:1]: For this iteration, business has told us to assume its English but noted this down as something we need to look at in the next iteration

**Review by stevenckwong — APPROVED** [review]
This is approved

**Pull request closed without merging**